### PR TITLE
Basic HTTP Server and Initial Boilerplate Files

### LINKED ISSUE #7: Stand up a basic http server with flask
See work done on the `example/flask` branch.

### PR DESCRIPTION
**Pull request by samgomena**
This PR closes #7

### CODE REVIEW

**Review by josephsPrograms — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,100 @@
...
+```
+
+#### Building with Docker
+
+```bash
+# Build the container setting ENV vars if needed
+docker build -t securemailbox/api:latest .
+
+# Run container in interactive mode
+docker run -it -p 8082:8082 securemailbox/api
```
Comment (campyne) [inline README.md]:
```suggestion
docker run -it --network host -e DATABASE_URL=$DATABASE_URL securemailbox/api
```

**Review by campyne — CHANGES_REQUESTED** [review]
I put in the change to the docker run command you came up with the other night.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,100 @@
...
+
+```bash
+# Run and daemonize a postgres instance exposing it on 127.0.0.1:5432
+# Docs: https://hub.docker.com/_/postgres/
+docker run -dp "5432:5432" -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_DB=securemailbox postgres:latest
+export DATABASE_URL=postgres://postgres:postgres@127.0.0.1:5432/securemailbox
+```
+
+#### Developing on Bare Metal
+
```
Comment (samgomena) [inline README.md:65]: Add docs about running on bare metal here.

File: securemailbox/__init__.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
```
Comment (samgomena) [inline securemailbox/__init__.py:4]: Check/ensure that python version is 3.6 +.

File: Pipfile [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+[dev-packages]
+
+[packages]
+flask = "*"
+flask-sqlalchemy = "*"
+flask-migrate = "*"
+psycopg2 = "*"
+
+[requires]
+python_version = "3.8"
```
Comment (samgomena) [inline Pipfile:15]: Update this to 3.6

File: securemailbox/__init__.py (hunk 3 above)
Comment (samgomena) [reply securemailbox/__init__.py:4]: Possibly log a warning since pipenv will hard pin it to a min version.

**Review by nvenetucci — APPROVED** [review]

**Review by Tom3424 — APPROVED** [review]

**Review by xingchengyusi — APPROVED** [review]

**Review by andrewbespaly — APPROVED** [review]

**Pull request merged**